### PR TITLE
Adding missing sequence to migration scripts

### LINKED ISSUE #3356: Missing sequence for `customuserattribute` table in migration
When adding the `customuserattribute` table during migration, the sequence for the `id` is missing (needed by e.g. PostgreSQL):
https://github.com/privacyidea/privacyidea/blob/5cc92448833bf5de3bb0ab10cd669496cfcef7d0/migrations/versions/888b56ed5dcb_.py#L20
We should check if other tables added through migration scripts are missing the sequence definition.

### PR DESCRIPTION
**Pull request by jona-samuel**
The script is missing the downgrade options

### CODE REVIEW

File: migrations/versions/a28f2733897b_.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+"""Sequence for cusstomuserattribute, tokengroup and tokentokengroup
```
Comment (cornelinux) [inline migrations/versions/a28f2733897b_.py:1]: Please add a version prefix here. I think it is "v3.8: ...."

File: migrations/versions/a28f2733897b_.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    if dialect_supports_sequences():
+        op.execute(CreateSequence(seq))
+
+
+def upgrade():
+    try:
+        seq = Sequence('customuserattribute_seq')
+        try:
+            create_seq(seq)
+        except Exception as _e:
```
Comment (cornelinux) [inline migrations/versions/a28f2733897b_.py:32]: I think in this case we can skipt the inner exception and simpy to

~~~~python
try:
    seq = Sequence(...)
    create_seq(seq)
except Exeption as exx:
    print(exx)
~~~~

in all cases.

**Review by cornelinux — CHANGES_REQUESTED** [review]

File: migrations/versions/a28f2733897b_.py (hunk 1 above)
Comment (jona-samuel) [reply migrations/versions/a28f2733897b_.py]: 79cdc78

File: migrations/versions/a28f2733897b_.py (hunk 2 above)
Comment (jona-samuel) [reply migrations/versions/a28f2733897b_.py]: 1b2bcbe

**Review by cornelinux — APPROVED** [review]

**Comment by Ma27** [comment]
This is actually causing issues when trying to upgrade to 3.8, see also https://github.com/privacyidea/privacyidea/issues/3447

**Pull request merged**